### PR TITLE
Add mhcflurry support

### LINKED ISSUE #42: support mhcflurry

### PR DESCRIPTION
**Pull request by julia326**
Fixes #42

### CODE REVIEW

File: mhctools/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     IedbSMM_PMBEC,
     IedbNetMHCIIpan,
 )
+from .mhcflurry import MHCFlurry
```
Comment (timodonnell) [inline mhctools/__init__.py]: nit: we've been calling it `MHCflurry` (lowercase F)

File: mhctools/mhcflurry.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        predictor = Class1AffinityPredictor.load(models_dir=self.models_dir)
+        encodable_sequences = EncodableSequences.create(peptides)
+        for allele in self.alleles:
+            predictions = predictor.predict(encodable_sequences, allele=allele)
+            for (i, peptide) in enumerate(peptides):
+                binding_prediction = BindingPrediction(
+                    allele=allele,
+                    peptide=peptide,
+                    affinity=predictions[i],
+                    percentile_rank=0.0,  # TODO: grab percentile rank when it's available
```
Comment (timodonnell) [inline mhctools/mhcflurry.py]: can this be set to numpy.nan instead of 0?

File: mhctools/mhcflurry.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+from mhcflurry import Class1AffinityPredictor
+from mhcflurry.encodable_sequences import EncodableSequences
+
+logger = logging.getLogger(__name__)
+
+
+class MHCFlurry(BasePredictor):
+    """
+    Wrapper around MHCFlurry. Assumes you've installed it and downloaded datasets and trained  MHCFlurry: see
```
Comment (timodonnell) [inline mhctools/mhcflurry.py]: If 'mhcflurry' is added as a requirement to mhctools I don't think you need to say "assumes you've instealled it". Also in any case no need to say " and trained  MHCFlurry": if they've downloaded datasets they don't need to train anything.

**Comment by timodonnell** [comment]
Left some very minor comments, LGTM!

**Comment by ihodes** [comment]
This is so awesome. We can use this for our lung run, too. 

One small request; could you update the README?

File: mhctools/__init__.py (hunk 1 above)
Comment (julia326) [reply mhctools/__init__.py]: done

File: mhctools/mhcflurry.py (hunk 2 above)
Comment (julia326) [reply mhctools/mhcflurry.py]: the BindingPrediction constructor wants an actual value here, seems like we avoid nans and Nones in other mhctools predictors - would you want to relax the restriction or have a nonsense value here?

Comment (timodonnell) [reply mhctools/mhcflurry.py]: I'd say better to relax the restriction and use `None` or `numpy.nan`. Don't want anyone accidentally using this value

**Pull request merged**